### ISSUE TITLE
Longer entry/match-object property names

### ISSUE THREAD

**Issue by stcruy** [vsmjs/vsm-dictionary#5]
Rename them like this:
```
- i --> id
- d --> dictID  (or dictId, or dictid, or dict?)
- x --> descr   (or expl, or def?)
- t --> terms
  - s --> str
  - y --> style
  - x --> descr  (or see above)
- z --> z
- w --> type   // } (used by match-objects only)
- s --> str    // }  "
- y --> style  // }  "
- tdel -> termsDel  // } (used by 'entryLike'-objects only)
- zdel -> zDel      // }  "
```

**Comment by bblodfon** [comment]
They seem more readable/understandable now :)
For the dictionary id I also prefer `dictID` and not `dictId` or the others.
For the description `x` value, well, `descr` vs `def` is difficult to answer, but I would go with the first one even though it is two chars longer.

**Comment by stcruy** [comment]
Thank you for the feedback! It's going on the right path then :)
I've picked `descr` too, because the word 'definition' sounds a bit too rigid (like mathematics), while 'description' is more in line with VSM's philosophy: that real-world concepts can often not be described with exactness.
I already made the major replacements. I'll commit them when I finished revising some more.

**Comment by stcruy** [comment]
Implemented in 79d8488567ab085fd08d7d2053c69bbc48bfa087.
